### PR TITLE
fix: dependabot with yarn2

### PR DESCRIPTION
**Pull request by tomfrenken**
Please provide a description of what your change does and why it is needed.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -4,5 +4,7 @@ updates:
   directory: "/"
   schedule:
     interval: daily
-    time: "01:00"
+    time: "09:14"
```
Comment (jjtang1985) [inline .github/dependabot.yml:7]:
```suggestion
    time: "08:40"
```
I would like to see whether this works and don't want to wait for another 34 min;)

**Comment by jjtang1985** [comment]
> Scheduled workflows run on the latest commit on the default or base branch.

I guess without merging to the main branch, the schedule won't work.
Here is the [doc](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#scheduled-events).

**Comment by tomfrenken** [comment]
Indeed, I've tested it on forks this far, but working with dependabot and Github actions on a fork is rather difficult because forks can't use secrets, and also the default token `GITHUB_TOKEN` which you usually use to authenticate a workflow has read-only permissions on a fork, so I wasn't able to push with a Github action to a branch.

**Comment by tomfrenken** [comment]
Also, the when you schedule a specific time, you are essentially still put in a queue before dependabot actually starts working, which delays it pretty much

**Comment by tomfrenken** [comment]
Okay I ran a modified version to be able to run on forks here:
https://github.com/tomfrenken/cloud-sdk-js/pull/1
seems to be working as intended

**Review by jjtang1985 — APPROVED** [review]
LGTM.
For the existing dependabot PR, please consider some follow up tests by e.g., making some dummy commits without real changes.

**Comment by jjtang1985** [comment]
Example git command will be `git commit --allow-empty -m "test"`.

**Pull request merged**